drivers/financial/spec/financial_spec.rb

```diff
@@ -0,0 +1,3 @@
+# GrdaWarehouse::CustomImports::Config.create(s3_prefix: "#{Rails.env}/#{Financial::Import.import_prefix}/", data_source_id: 2, s3_region: 'us-east-1', s3_bucket: 'financial-bucket', active: true, import_type: 'Financial::Import', user: User.system_user, import_hour: 6, s3_access_key_id: 'local_access_key', s3_secret_access_key: 'local_secret_key')
```

Comment:
Are we missing a commit here?

Reply:
oh, no.  I meant to add some tests, but ended up just testing things manually.  I'll add a TODO for future development.